src/xf86libinput.c

```diff
@@ -5641,7 +5641,8 @@ LibinputInitTapProperty(DeviceIntPtr dev,
 			struct xf86libinput *driver_data,
 			struct libinput_device *device)
 {
-	BOOL tap = driver_data->options.tapping;
+	// By default tapping property config is true
+	BOOL tap = driver_data->options.tapping ? driver_data->options.tapping : TRUE;
```

Comment:
This is not the correct way to set the default value. This just ignores the value the user sets. The default value should be set in `xf86libinput_parse_tap_option`. Take a look at how it's done in `xf86libinput_parse_scroll_pixel_distance_option`. I dont have a laptop anymore so I cant do any of that myself.